vrp-cli/src/lib.rs

```diff
@@ -525,9 +530,17 @@ pub fn get_solution_serialized(problem: Arc<CoreProblem>, config: Config) -> Res
 
     let mut writer = BufWriter::new(Vec::new());
     if let Some(metrics) = metrics {
-        (&solution, cost, &metrics).write_pragmatic_json(&problem, &mut writer)?;
+        if geojson {
```

Comment:
I'm getting clippy error on this block: `error: this `else { if .. }` block can be collapsed...`

Please try to run `cargo clippy --all-features --tests -- -D warnings` on your changes.